copier/main.py

```diff
@@ -426,8 +426,21 @@ def _render_file(self, src_abspath: Path) -> None:
         if dst_relpath is None:
             return
         if src_abspath.name.endswith(self.template.templates_suffix):
-            tpl = self.jinja_env.get_template(str(src_relpath))
-            new_content = tpl.render(**self._render_context()).encode()
+            try:
+                tpl = self.jinja_env.get_template(src_relpath)
+            except UnicodeDecodeError:
+                if self.template.templates_suffix:
+                    # suffix is not emtpy, re-raise
+                    print(
+                        colors.warn
+                        | f"{src_relpath} could not be read as a Jinja2 template, falling back to a simply copy",
+                        file=sys.stderr,
+                    )
```

Comment:
Wasn't sure about how to log that (related to #368). Let me know if there's a better way, or if the phrasing should be improved/shortened.

Reply:
In this case the message is a lie, so just remove it and let the exception raise. The designer deliberately added `.jinja` to a binary file, and that's a bug on his side.

Reply:
Erm, of course, I'm re-raising, not copying the file, the message is completely wrong! Thanks for noticing :sweat_smile:

Reply:
I moved the message below, where it actually belongs.

Reply:
But maybe you'd prefer having no messages at all. Just let me know and I'll remove it.